apps/web/components/ui/input-otp.tsx

```
@@ -54,18 +54,18 @@ const InputOTPSlot = React.forwardRef<
-InputOTPSlot.displayName = "InputOTPSlot"
+  );
+});
+InputOTPSlot.displayName = 'InputOTPSlot';
 
 const InputOTPSeparator = React.forwardRef<
-  React.ElementRef<"div">,
-  React.ComponentPropsWithoutRef<"div">
+  React.ElementRef<'div'>,
+  React.ComponentPropsWithoutRef<'div'>
 >(({ ...props }, ref) => (
   <div ref={ref} role="separator" {...props}>
```

Biome (2.1.2)

[error] The HTML element with the interactive role "separator" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "separator".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)